packages/@react-native-windows/tester/overrides.json

```diff
@@ -61,11 +61,25 @@
       "baseHash": "c4d61d6e76488c48b8256e6abc9bb38ef8e54c44",
       "issue": 5688
     },
+    {
+      "type": "patch",
+      "file": "src/js/examples/TextInput/TextInputSharedExamples.windows.js",
+      "baseFile": "packages/rn-tester/js/examples/TextInput/TextInputSharedExamples.js",
+      "baseHash": "55efb5c3f34963215650859b8a1bf3cbb5fc6f54",
+      "issue": 13303
+    },
+    {
+      "type": "patch",
+      "file": "src/js/examples/TurboModule/SampleTurboModuleExample.windows.js",
+      "baseFile": "packages/rn-tester/js/examples/TurboModule/SampleTurboModuleExample.js",
+      "baseHash": "6e532e2e2fab2fd61b864f6ebe64d7725061043d",
+      "issue": 0
```

Comment:
Is there an issue to track this?

Reply:
The functionality should be android only so we shouldn't need a tracking issue, unless we decide to implement